language/move-prover/spec-lang/src/translate.rs

```diff
@@ -960,24 +987,38 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
                     );
                 }
             }
-            let translated = et.translate_exp(exp, &BOOL_TYPE);
+            let expected_type = match kind {
```

Comment:
xclippy will probably ask you to make an if let out of this (and Ci will fail if not)

Reply:
Clippy does not appear to be complaining.